packages/__cxstarter__/src/shadow/@bodiless/cx-flowcontainer/FlowContainer.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { asFluidToken } from '@bodiless/cx-elements';
+import { cxFlowContainer } from '@bodiless/cx-flowcontainer';
+import { cxEditorsFlowContainer } from '@bodiless/cx-editors';
+
+const Default = asFluidToken({
+  ...cxFlowContainer.Default,
+  Components: {
```

Comment:
Are you including this just as an example?  You seem to be removing other components (Like images and the nested flow container) - is this intentional?